runbotics/runbotics-orchestrator-ui/src/src-app/components/tables/HistoryTable/HistoryTable.types.ts

```diff
@@ -5,13 +5,15 @@ import { SxProps, Theme } from '@mui/system';
 import { Row } from 'react-table';
 
 import { InstanceExtendedWithSubprocesses, ProcessInstanceRequestCriteria } from '#src-app/store/slices/ProcessInstance';
-
-export interface getSubprocessesResponse { 
+import { Page } from '#src-app/utils/types/page';
+export interface GetSubprocessesResponse {
     type: string;
-    payload: InstanceExtendedWithSubprocesses[];
+    payload: Page<InstanceExtendedWithSubprocesses[]>
     meta: {
         arg: {
             processInstanceId: string;
+            page?: number;
+            size?: number;
```

Comment:
I'm not sure if you still need page and size properties in meta.arg

Reply:
agree, seems it is not used anywhere as .meta.arg